coral/coral/pkg/graphs/resource_models/License.json

```diff
@@ -533,49 +615,273 @@
                     "nodegroup_id": "4f0f655c-48cf-11ee-8e4e-0242ac140007",
                     "sortorder": null,
                     "visible": true
+                }
+            ],
+            "cards_x_nodes_x_widgets": [
+                {
+                    "card_id": "cd6d687b-c9e8-41e1-9a9f-9967dd3732ab",
+                    "config": {
+                        "dateFormat": "YYYY-MM-DD",
+                        "defaultValue": "",
+                        "label": "Report Submitted",
+                        "maxDate": false,
+                        "minDate": false,
+                        "placeholder": "Enter date",
+                        "viewMode": "days"
+                    },
+                    "id": "712c03c1-a016-406e-9e55-2138a9220e6e",
+                    "label": {
+                        "en": "Report Submitted"
+                    },
+                    "node_id": "33dab998-6121-11ee-9fd1-0242ac120003",
+                    "sortorder": 0,
+                    "visible": true,
+                    "widget_id": "10000000-0000-0000-0000-000000000004"
                 },
                 {
-                    "active": true,
```

Comment:
This will need to be merged locally using the json diff tool to avoid conflicts/remove something.

Reply:
This was merged locally with json_diff